index.html

```diff
@@ -27,9 +27,115 @@
+            <div><ul id="l1">
+                 <li id="lang">HTML</li><li id="lang">CSS</li><li id="lang">JavaScript</li>
+            </ul></div>
+            <div id="stp">See this project
+                <img id="arrow" src="media/arrow.png">
+            </div>
+        </div>
+        <div class="flex-container">
+           <div><img id="rectangle2" src="media/Rectangle 2.png"></div>
+            <div id="st">Project name goes here</div>
+            <div><ul id="l1">
+                <li id="lang">HTML</li><li id="lang">CSS</li><li id="lang">JavaScript</li>
+           </ul></div>
+           <div id="stp">See this project
+            <img id="arrow" src="media/arrow.png">
+            </div>
+        </div>
+        <div class="flex-container">
+            <div><img id="rectangle2" src="media/Rectangle 3.png"></div>
+            <div id="st">Project name goes here</div>
+            <div><ul id="l1">
+                <li id="lang">HTML</li><li id="lang">CSS</li><li id="lang">JavaScript</li>
+           </ul></div>
+           <div id="stp">See this project
+            <img id="arrow" src="media/arrow.png">
+            </div>
+        </div>
+        <div class="flex-container">
+            <div><img id="rectangle2" src="media/Rectangle 4.png"></div>
+            <div id="st">Project name goes here</div>
+            <div><ul id="l1">
+                <li id="lang">HTML</li><li id="lang">CSS</li><li id="lang">JavaScript</li>
+           </ul></div>
+           <div id="stp">See this project
+            <img id="arrow" src="media/arrow.png">
+            </div>
+        </div>
+        <div class="flex-container">
+            <div><img id="rectangle2" src="media/Rectangle 5.png"></div>
+            <div id="st">Project name goes here</div>
+            <div><ul id="l1">
+                <li id="lang">HTML</li><li id="lang">CSS</li><li id="lang">JavaScript</li>
+           </ul></div>
+           <div id="stp">See this project
+            <img id="arrow" src="media/arrow.png">
+            </div>
+        </div>
+        <div class="flex-container">
+            <div><img id="rectangle2" src="media/Rectangle 6.png"></div>
+            <div id="st">Project name goes here</div>
+            <div><ul id="l1">
+                <li id="lang">HTML</li><li id="lang">CSS</li><li id="lang">JavaScript</li>
+           </ul></div>
+           <div id="stp">See this project
+            <img id="arrow" src="media/arrow.png">
+           </div>
+        </div>
+        <div  class="flex-container2">
+            <div id="abt-t">
+                About
+            </div>
+            <div id="abt">
+                Lorem ipsum dolor sit amet consectetur, adipisicing elit. At consequatur eaque, sequi, dignissimos distinctio minus cupiditate, perferendis praesentium debitis tenetur odio. Totam accusamus atque quo aut! Numquam sapiente ex dolores?
+            </div>
+            <div id="cv">Get my CV
```

Comment:
- [ ] kindly add a link placeholder to the `get my CV` button. This will assist in future to take you to your cv on click